mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -215,9 +216,16 @@ const PhaseOneBasicGamePlay = ({
                                 {("0" + Math.floor(currentTime % 60)).slice(-2)}
                             </Text>
                         </View>
+
                     </>
                 ) : null}
             </LinearGradient>
+            <LinearGradient colors={["rgba(62, 0, 172, 1)", "rgba(98, 0, 204, 1)"]}
+                style={styles.headerContainer}
+                start={{ x: 0, y: 1 }}
+                end={{ x: 1, y: 1 }}>
+
+            </LinearGradient>
```

Comment:
Looks to be duplicate of the existing gradient, any reason we need two?

Reply:
(for all above items): Some of the above changes were made to accommodate for the changes caused by removing scale but I now realize this makes much less sense. I reverted most of the changes since I haven't been able to test on multiple devices yet to be on the safe side. I did notice a padding issue on the wrong answer explanation cards though and pushed this from another branch along with the scales added back in.

Reply:
Check the last part in [this line](https://github.com/rightoneducation/righton-app/blob/dev/mobile/package.json#L10). You should be able to easily change the name of the device there to anything else and run the app on that simulator. You can open the app in Xcode, by opening RightOn.xcworkspace under `ios` folder. Then check the top for name of simulators you have.

![Screen Shot 2022-12-15 at 6 59 31 PM](https://user-images.githubusercontent.com/2628592/207991776-784754e9-40d0-415e-96fc-84a043345769.png)